### ISSUE TITLE
Lexical editor forcibly in readonly mode

### ISSUE THREAD

**Issue by stuckinsnow** [payloadcms/payload#9363]
### Describe the Bug

Lexical editor is in readonly mode, the toolbar at the top doesn't show, and it is not sparking the joy I had assumed it would spark. I don't have any type errors anywhere, no other errors. Types build correctly, import map works, etc.  

### Link to the code that reproduces this issue

NA

### Reproduction Steps

1. Made a blank template, using v127
2. Put lexical inside of tabs
3. Updated to 3.0.1 

### Which area(s) are affected? (Select all that apply)

area: core

### Environment Info

```text
NA
```

**Comment by jakehopking** [comment]
I'm having the exact same issue. Some instances of the lexical editor, with the exact same config yet in different collections are greyed out / read-only state.

Nightmare as it now in prod and the editors are stuck.

![Image](https://github.com/user-attachments/assets/86461331-c34c-45ac-bbe6-ae0ba36bfcc0)
![Image](https://github.com/user-attachments/assets/d3c38764-7559-4693-864e-4c11dc355b9e)

**Comment by jakehopking** [comment]
Another issue, also within tabs is relationship selects are randomly blank.

You can see reusableContent is working in some places, and not others.

![Image](https://github.com/user-attachments/assets/7c71ee71-2c4e-4bf7-a222-f65cea8e86de)

![Image](https://github.com/user-attachments/assets/b744238e-2167-4539-9b51-09fb21cf06e6)

**Comment by bowendevteam** [comment]
to provide some more context here, I did some testing and it appears as though beta.132 broke this. I downgrade to beta.131 and the lexical editors are no longer read only mode. Also a ton of other bugs seemed to be resolved

**Comment by jakehopking** [comment]
I have done the same as @bowendevteam. Downgrading to 131 has solved the lexical editor readonly issue. 
However the relationship issue persists, so must have been introduced in the migration script...?

**Comment by jmikrut** [comment]
Hey all, we are actively looking into this. @jakehopking — are your tabs named, or unnamed? Can you provide us with an exact structure of fields where your rich text editor is either blank or rendering as read-only?

**Comment by jakehopking** [comment]
`heroArticle.ts`

```ts
import type { Field } from 'payload'

import {
  FixedToolbarFeature,
  HeadingFeature,
  InlineToolbarFeature,
  lexicalEditor,
} from '@payloadcms/richtext-lexical'

export const heroArticle: Field = {
  name: 'hero',
  type: 'group',
  fields: [
    {
      name: 'type',
      type: 'select',
      defaultValue: 'articleHorizontal',
      label: 'Type',
      options: [
        {
          label: 'Article Vertical',
          value: 'articleVertical',
        },
        {
          label: 'Article Horizontal',
          value: 'articleHorizontal',
        },
      ],
      admin: {
        isClearable: false,
      },
      required: true,
    },
    {
      name: 'richTextHeading',
      type: 'richText',
      label: 'Heading',
      required: true,
      admin: {
        description: 'The heading that will be displayed in the hero section.',
      },
      editor: lexicalEditor({
        features: ({ rootFeatures }) => {
          return [
            ...rootFeatures,
            HeadingFeature({ enabledHeadingSizes: ['h1'] }),
            FixedToolbarFeature(),
            InlineToolbarFeature(),
          ]
        },
      }),
    },
    {
      name: 'richTextIntro',
      type: 'richText',
      label: 'Intro',
      required: true,
      editor: lexicalEditor({
        features: ({ rootFeatures }) => {
          return [
            ...rootFeatures,
            // HeadingFeature({ enabledHeadingSizes: [] }),
            FixedToolbarFeature(),
            InlineToolbarFeature(),
          ]
        },
      }),
    },
    {
      name: 'media',
      type: 'upload',
      admin: {
        description:
          'The image that will be displayed in the hero section. Note: To specify an images to use within carousels, cards, and SEO, see the `Carousel`, `Card`, and `SEO` tabs respectively.',
      },
      relationTo: 'media',
      required: true,
    },
    {
      name: 'showCaption',
      type: 'checkbox',
      defaultValue: false,
      label: 'Show Media Caption',
      admin: {
        description: 'Show the caption for the media. Note: Caption is set in the Media Library.',
      },
    },
  ],
  label: false,
}
```

is imported into article `index.ts`

```ts
import type { CollectionConfig } from 'payload'
import { authenticated } from '@payload/access/authenticated'
import { authenticatedOrPublished } from '@payload/access/authenticatedOrPublished'
import { slugField } from '@payload/fields/slug'
import { heroArticle } from '@payload/fields/heroArticle'
import { publishedAt } from '@payload/fields/publishedAt'
import { collectionNameField } from '@payload/fields/collectionName'
import { populatePublishedAt } from '@payload/hooks/populatePublishedAt'
import { generatePreviewPath } from '@payload/utilities/generatePreviewPath'
import { CollectionSlugs } from '@tokens/slugs'
import { revalidateArticle } from './hooks/revalidateArticle'
import { themeField } from '@payload/fields/theme'
import { categoriesField } from '@payload/fields/categories'
import { tagsField } from '@payload/fields/tags'
import { authorsField } from '@payload/fields/authors'
import { populateAuthors } from '@payload/hooks/populateAuthors'
import { ArticleMediaBlock } from '@payload/blocks/ArticleMediaBlock'
import { ArticleQuoteBlock } from '@payload/blocks/ArticleQuoteBlock'
import { ArticleButtonBlock } from '@payload/blocks/ArticleButtonBlock'
import { FormBlock } from '@payload/blocks/Form'
import { ReusableContent } from '@payload/blocks/ReusableContent'
import { LinksGridBlock } from '@payload/blocks/LinksGrid'
import { commonTabsCardFields, commonTabsCarouselFields } from '@payload/fields/commonTabs'
import {
  BlocksFeature,
  FixedToolbarFeature,
  HeadingFeature,
  HorizontalRuleFeature,
  InlineToolbarFeature,
  lexicalEditor,
} from '@payloadcms/richtext-lexical'
import {
  MetaDescriptionField,
  MetaImageField,
  MetaTitleField,
  OverviewField,
  PreviewField,
} from '@payloadcms/plugin-seo/fields'
import { revalidateCollectionAndSlug } from '@payload/hooks/revalidateCollectionAndSlug'
import { ArticleVideoBlock } from '@payload/blocks/ArticleVideoBlock'
import { deleteCollectionAndSlug } from '@payload/hooks/deleteCollectionAndSlug'
import { deleteArticle } from './hooks/deleteArticle'

export const Articles: CollectionConfig = {
  slug: CollectionSlugs.articles,
  access: {
    create: authenticated,
    delete: authenticated,
    read: authenticatedOrPublished,
    update: authenticated,
  },
  admin: {
    useAsTitle: 'title',
    defaultColumns: ['title', 'slug', 'publishedAt', 'updatedAt'],
    livePreview: {
      url: ({ data }) => {
        const path = generatePreviewPath({
          // path: `${RouterPaths.articles}${typeof data?.slug === 'string' ? data.slug : ''}`,
          slug: `${typeof data?.slug === 'string' ? data.slug : ''}`,
          collection: 'articles',
        })
        return `${process.env.NEXT_PUBLIC_SERVER_URL}${path}`
      },
    },
    preview: (data) =>
      generatePreviewPath({
        slug: `${typeof data?.slug === 'string' ? data.slug : ''}`,
        collection: 'articles',
      }),
  },
  fields: [
    {
      name: 'title',
      type: 'text',
      required: true,
    },
    {
      name: 'description',
      type: 'textarea',
      required: true,
      admin: {
        description:
          'This description, or abstract, is used (unless overridden) for carousel descriptions, card descriptions, and SEO descriptions. To override the aforementioned descriptions, see the `Carousel`, `Card`, and `SEO` tabs respectively.',
      },
    },
    {
      type: 'tabs',
      tabs: [
        {
          label: 'Hero',
          fields: [heroArticle],
        },
        {
          label: 'Content',
          fields: [
            {
              name: 'content',
              type: 'richText',
              editor: lexicalEditor({
                features: ({ rootFeatures }) => {
                  return [
                    ...rootFeatures,
                    HeadingFeature({ enabledHeadingSizes: ['h2', 'h3', 'h4', 'h5'] }),
                    BlocksFeature({
                      blocks: [
                        ArticleMediaBlock,
                        ArticleVideoBlock,
                        ArticleQuoteBlock,
                        ArticleButtonBlock,
                        FormBlock,
                      ],
                    }),
                    FixedToolbarFeature(),
                    InlineToolbarFeature(),
                    HorizontalRuleFeature(),
                  ]
                },
              }),
              label: false,
              required: true,
            },
          ],
        },
        {
          label: 'Blocks',
          fields: [
            {
              name: 'layout',
              type: 'blocks',
              blocks: [ReusableContent, LinksGridBlock],
              admin: {
                description:
                  'Additional blocks can be added here. These are placed at the bottom of the article.',
              },
            },
          ],
        },
        {
          label: 'Carousel',
          description: 'This tab is used to set how this page appears when used within carousels.',
          fields: [...commonTabsCarouselFields],
        },
        {
          label: 'Card',
          description: 'This tab is used to set how this page appears when used within cards.',
          fields: [...commonTabsCardFields],
        },
        {
          label: 'Meta',
          description: "This tab is used to set the article's meta settings.",
          fields: [
            {
              name: 'relatedArticles',
              type: 'relationship',
              admin: {
                position: 'sidebar',
              },
              filterOptions: ({ id }) => {
                return {
                  id: {
                    not_in: [id],
                  },
                }
              },
              hasMany: true,
              relationTo: CollectionSlugs.articles,
            },
            categoriesField,
            tagsField,
            themeField({}),
          ],
        },
        {
          label: 'SEO',
          name: 'meta',
          fields: [
            OverviewField({
              titlePath: 'meta.title',
              descriptionPath: 'meta.description',
              imagePath: 'meta.image',
            }),
            MetaTitleField({
              hasGenerateFn: true,
            }),
            MetaImageField({
              relationTo: 'media',
            }),

            MetaDescriptionField({}),
            PreviewField({
              // if the `generateUrl` function is configured
              hasGenerateFn: true,

              // field paths to match the target field for data
              titlePath: 'meta.title',
              descriptionPath: 'meta.description',
            }),
          ],
        },
      ],
    },
    publishedAt,
    slugField(),
    authorsField,
    // This field is only used to populate the user data via the `populateAuthors` hook
    // This is because the `user` collection has access control locked to protect user privacy
    // GraphQL will also not return mutated user data that differs from the underlying schema
    {
      name: 'populatedAuthors',
      type: 'array',
      access: {
        update: () => false,
      },
      admin: {
        disabled: true,
        readOnly: true,
      },
      fields: [
        {
          name: 'id',
          type: 'text',
        },
        {
          name: 'name',
          type: 'text',
        },
      ],
    },
    collectionNameField(CollectionSlugs.articles),
  ],
  hooks: {
    afterChange: [revalidateCollectionAndSlug, revalidateArticle],
    afterDelete: [deleteCollectionAndSlug, deleteArticle],
    afterRead: [populateAuthors],
    beforeChange: [populatePublishedAt],
  },
  versions: {
    drafts: {
      autosave: {
        interval: 100, // We set this interval for optimal live preview
      },
    },
    maxPerDoc: 50,
  },
}
```

**Comment by jakehopking** [comment]
Issue fixed thanks to Philip!

```ts
      name: 'populatedAuthors',
      type: 'array',
      access: {
        update: () => false, // <--- change this to true until a fix is released
      },
```

**Comment by jmikrut** [comment]
Fix incoming first thing tomorrow - thank you all for helping reproduce!
